src/cms/views/regions/region_list_view.py

```diff
@@ -42,11 +43,40 @@ def get(self, request, *args, **kwargs):
         :return: The rendered template response
         :rtype: ~django.template.response.TemplateResponse
         """
+
         regions = Region.objects.all()
+        query = None
+
+        search_data = kwargs.get("search_data")
+        if search_data:
+            search_form = ObjectSearchForm(search_data)
```

Comment:
If `ObjectSearchForm` could deal with a `None` parameter, then the previous check (in some other places as well) would not be necessary.

Reply:
Since `ObjectSearchForm` is just a regular Django form, passing in `None` as data is not a problem (in fact, it's a keyword parameter, so it makes no difference if you call `ObjectSearchForm()`, `ObjectSearchForm(None)` or `ObjectSearchForm(data=None)`).